packages/evm/test/voting/crvoting-commit.js

```diff
@@ -298,6 +302,10 @@ contract('CRVoting', ([_, voter, someone, delegate, governor]) => {
           })
 
           context('when the sender does not have permission', () => {
+            beforeEach('revoke role', async () => {
+              await controller.grant(roleId(delegate, 'commit'), sender, { from: governor })
```

Comment:
Should this be `revoke()` instead? Although it's a bit weird, shouldn't this make the test fail?

Reply:
It failed on my computer, this is weird